eslintrc.json

```diff
@@ -48,22 +49,23 @@
     "no-void": 2,
     "no-with": 2,
     "prefer-promise-reject-errors": 2,
-    "require-await": 2,
+    "require-await": 0,
     "vars-on-top": 2,
     "wrap-iife": 2,
     "no-delete-var": 2,
-    "no-shadow": 2,
+    "no-shadow": 0,
     "no-shadow-restricted-names": 2,
     "no-undef-init": 2,
-    "no-use-before-define": [2, { "functions": true }],
+    "no-use-before-define": 0,
     "callback-return": [2, ["callback", "cb", "next", "done"] ],
     "global-require": 2,
     "handle-callback-err": 2,
     "no-new-require": 2,
     "no-path-concat": 2,
     "no-process-env": 2,
     "no-process-exit": 2,
-    "no-sync": 1,
+    "no-magic-numbers": 0,
+    "no-sync": 0,
```

Comment:
I prefer leaving sync as an error. If you really want to use a sync method, I prefer the programmer silences explicitly the line of code. Using it in the wrong line of code could be dangerous.